README.md

```diff
@@ -23,14 +24,32 @@ A curated list of awesome projects built with [Hyperapp](https://github.com/hype
 - [hypernews](https://github.com/traducer/hypernews) - Hacker News clone with Hyperapp.
 
 ## Libraries and Mixins
+- [hyperapp-deepupdate](https://github.com/NoobLad/hyperapp-deepupdate) - A utility for updating deeply nested state.
+- [hyperapp-dot-notation-reducer](https://github.com/alber70g/hyperapp-dot-notation-reducer) - Allows actions to return an object with a path as a path as a property.
```

Comment:
`...with a path as a path as a property` is this intentional?

Reply:
There is only one person that can answer that! @alber70g 😄

Reply:
ah shoot :/ will fix. Been super busy at work the last few days if seems I am slow with this.

Reply:
thanks @RikuVan 🙏